### PR TITLE
Deploy proposals only when option want_ is equal to 1

### PR DESCRIPTION
**Pull request by mjura**
We would like to be sure that proposal will be applied only when
assigned option want_ is equal to 1.

For current condtions it is enough that for example
    export want_barbican=0

and condition

    [[ $want_barbican ]] || return

will not work

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3330,7 +3330,7 @@ function deploy_single_proposal()
             fi
             ;;
         swift)
-            [[ $deployswift ]] || return
+            [[ $deployswift = 1 ]] || return
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: this is an unrelated change I think

Comment (mjura) [inline scripts/qa_crowbarsetup.sh]: Reverted this change

**Comment by mjura** [comment]
> the alternative would be to change the defaults for the other to '' and 1, respecticvely. I have no strong opinion which one is better.

Problem is that in current condition check, [[ $want_murano ]] this will be true even when want_murano will be 0, 1, 2, 3, 4 ... I think would be better to have explicit $want_murano and others equal to 1
